app/server.py

```diff
@@ -90,18 +106,22 @@ def start_bot_process(self, bot_uuid):
         self._get_questions_queue(bot_uuid)
 
     def start_garbage_collector(self):
-        Timer(5*60.0, self.garbage_collector).start()
+        Timer(60.0, self.garbage_collector).start()
```

Comment:
I think that `time` could be on settings file as parameter

Reply:
I did had solved in next pull request, any problem if it come in next PR?